_README.txt

```diff
@@ -68,6 +68,16 @@ COMPILATION/SETUP:
    - press 4 to toggle stiffness between normal and super elastic
    - press 5 to toggle between bullet time (1/4th of real time) and real time 
    - press 6 to repeat precomputation
+
+COMPILATION CMAKE:
+ - prerequisites:
+    - Install CMake version 3.22.1 or higher
+    - Install a compiler of your choice Visual studio build tools, gcc and minGW should work
+ - use an IDE to import the project as cmake project and run it or use command line
+    - CMAKE COMMANDLINE:
+        - make a folder called build in the project directory
+        - open a terminal in the folder and call `cmake ../`
+        - now build the generated build folder by cmake with your preferred command line tool.

```

Comment:
This should be the standard way now, as it is extremely convenient, so feel free to put this on top.

I will revise the `README` to reflect all changes after both your PRs have been merged (and turn it into a proper gitlab-readable README.md).

Reply:
I put it above the normal compilation. I didn't get the CUDA branch to compile as of yet since some functions have been deprecated in the newer versions of CUDA and for some reason my GPU did not agree with compiling an older version from 2018.  I will first have to focus on exams for now and an assignment for a course I am working on, but then I'll retry.